docs/versioned_docs/version-v1.0.x/installation.md

```diff
@@ -7,9 +7,9 @@ title: Installation
 To install Eraser with the manifest file, run the following command:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v1.0.0/deploy/eraser.yaml
+kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/master/deploy/eraser.yaml
```

Comment:
why are we changing to master? this branch doesn't exist

Reply:
Not sure how that happened, it's a mistake. I'll revert it